forms-flow-data-layer/src/graphql/schema/formio_form_schema.py

```diff
@@ -11,7 +13,9 @@ class FormSchema:
     """
 
     id: str
-    name: str
+    name: Optional[str] = strawberry.field(
+        extensions=[RoleCheck(["designer"])]
```

Comment:
We dont have a designer role anymore right ? we should be checking against the new permissions.

Reply:
resolved